src/furo/theme/base.html

```diff
@@ -51,7 +51,7 @@
 
     {#- Theme-related stylesheets -#}
     {%- block theme_styles -%}
-    <link rel="stylesheet" href="{{ pathto('_static/' + style, 1) | e }}?state={{ furo_asset_hash }}">
+    <link rel="stylesheet" href="{{ furo_asset_hash(pathto('_static/' + style, 1) | e) }}">
```

Comment:
```suggestion
    <link rel="stylesheet" href="{{ furo_asset_hash(pathto('_static/' + style, 1)) | e }}">
```

Reply:
Sure. I have no idea what `e` is. It's not the most googlable thing.